robber/matchers/exception.py

```diff
@@ -0,0 +1,33 @@
+from robber import expect
+from robber.matchers.base import Base
+
+
+class ExceptionMatcher(Base):
+    """
+    expect(lambda: call_something(with_some_params)).to.throw(any_exception)
+    """
+    @property
+    def _raised(self):
+        raised = None
+
+        if not callable(self.actual):
+            raise TypeError('{actual} is not callable'.format(actual=self.actual))
+
+        try:
+            self.actual()
+        except BaseException as raised:
```

Comment:
Hard to say which kind of exception we should use in tests, so, I prefer to get the BaseException instead of Exception.